src/peft/tuners/vera/config.py

```diff
@@ -0,0 +1,123 @@
+        projection_prng_key (`int`):
+            Vera PRNG init key. Used for initialising vera_A and vera_B for new models, or when the checkpoint did not
+            include these projections.
+        target_modules (`Union[List[str],str]`): The names of the modules to apply Vera to.
+        projection_prng_key:
+            (`Optional[int]`): Vera PRNG init key. Used for initialising vera_A and vera_B for new models, or when the
+            checkpoint
+        did not include these projections.
+        save_projection:
+            (`bool`): Whether to save the vera_A / vera_B projections in the state dict alongside per layer lambda_b /
+            lambda_d weights. This will increase the size of the checkpoint, but guarantee that we can reload the
+            checkpoint on all system configurations.)
+        vera_dropout (`float`): The dropout probability for Vera layers.
+        d_initial (`float`): Initial init value for `vera_lambda_d` vector used when `init_vera_weights` is `True`.
+        fan_in_fan_out (`bool`): Set this to True if the layer to replace stores weight like (fan_in, fan_out).
+            For example, gpt-2 uses `Conv1D` which stores weights like (fan_in, fan_out) and hence this should be set
+            to `True`.
+        bias (`str`): Bias type for Vera. Can be 'none', 'all' or 'vera_only'. If 'all' or 'vera_only', the
+            corresponding biases will be updated during training. Be aware that this means that, even when disabling
+            the adapters, the model will not produce the same output as the base model would have without adaptation.
+        modules_to_save (`List[str]`):List of modules apart from Vera layers to be set as trainable
+            and saved in the final checkpoint.
+        layers_to_transform (`Union[List[int],int]`):
+            The layer indexes to transform, if this argument is specified, it will apply the Vera transformations on
+            the layer indexes that are specified in this list. If a single integer is passed, it will apply the Vera
+            transformations on the layer at this index.
+        layers_pattern (`str`):
+            The layer pattern name, used only if `layers_to_transform` is different from `None` and if the layer
+            pattern is not in the common layers pattern.
+    """
+
+    r: int = field(default=8, metadata={"help": "Vera attention dimension"})
+
+    target_modules: Optional[Union[List[str], str]] = field(
+        default=None,
+        metadata={
+            "help": "List of module names or regex expression of the module names to replace with Vera."
+            "For example, ['q', 'v'] or '.*decoder.*(SelfAttention|EncDecAttention).*(q|v)$' "
+        },
+    )
+    projection_prng_key: Optional[int] = field(
+        default=None,
+        metadata={
+            "help": "Vera PRNG init key. Used for initialising vera_A and vera_B for new models, or when the checkpoint did not include these projections."
```

Comment:
Done, just FYI you can add the flag `--experimental-string-processing` to `black` and it will handle strings too.

See https://stackoverflow.com/questions/68596593/how-to-automatically-break-long-string-constants-in-python-code-using-black-form